toolchain/check/context.h

```diff
@@ -101,8 +101,15 @@ class Context {
   // evaluation, either because it's newly created and entirely unused, or
   // because it's only used in a position that constant evaluation ignores, such
   // as a return slot.
-  auto ReplaceInstBeforeConstantUse(SemIR::InstId inst_id,
-                                    SemIR::LocationIdAndInst loc_id_and_inst)
+  auto ReplaceLocationIdAndInstBeforeConstantUse(
+      SemIR::InstId inst_id, SemIR::LocationIdAndInst loc_id_and_inst) -> void;
+
+  // Replaces the value of the instruction `inst_id` with `inst`.
```

Comment:
Is the value of an instruction ID an `Inst` (as this comment suggests) or a `LocationIdAndInst` (as the previous comment suggests)?

Reply:
The "value" is could mean either or both depending on context. I've dropped "the value" since it doesn't seem to help.